docs/requirements.txt

```diff
@@ -53,5 +69,7 @@ sphinxcontrib-qthelp==2.0.0
     # via sphinx
 sphinxcontrib-serializinghtml==2.0.0
     # via sphinx
-urllib3==2.5.0
+urllib3==2.3.0
     # via requests
+myst-parser
+Markdown>=3.2.1,<3.4
```

Comment:
Hoe ben je tot deze wijzigingen gekomen? Ik ben specifiek nieuwsgierig naar `Markdown`. We updaten dit bestand normaal gesproken namelijk niet handmatig maar met het commando `uv pip compile --output-file requirements.txt requirements.in`. Als ik dat doe dan komt `Markdown` niet in dit bestand terecht.

Ik denk dat het goed is om iets van een handleiding in deze map te zetten hoe je dependencies update. Daar ga ik wel achteraan.

Reply:
Ik heb volgens mij de requirements van een ander project gepakt, en het daaruit overgenomen. 
Ik weet alleen niet meer zo goed welke. Ik zie deze regel ook in ITA staan....

Reply:
Ik zie dat we daar de requirements.txt niet automatisch genereren. Ik genereer het bestand opnieuw en check het in. Kan jij dan kijken of de readthedocs nog werkt zoals verwacht?